--- src/fs/Kconfig.binfmt	2022-06-18 18:53:48.030702025 -0500
+++ src/fs/Kconfig.binfmt.orig	2022-06-18 18:53:42.577707991 -0500
@@ -160,6 +160,17 @@
 	  or /lib/ld.so) are still in a.out format, you will have to
 	  say Y here.
 
+config BINFMT_MACHO
+	bool "Enable Mach-O binary format support"
+	help
+		Mach-O, short for Mach object file format, is a file format for executables, object code, 
+	   shared libraries, dynamically-loaded code, and core dumps. It was developed to replace the a.out format.
+
+	   While the default format for Linux is ELF (Executable and Linkable Format), some Linux userland implementions
+	   might asume Mach-O support on the kernel, such as Utopia.
+
+	   If unsure about if this option should be turned on or not, is recommended to leave it up to your default configuration.
+
 config OSF4_COMPAT
 	bool "OSF/1 v4 readv/writev compatibility"
 	depends on ALPHA && BINFMT_AOUT
